src/ModelRegion.cpp

```diff
@@ -407,7 +412,11 @@ namespace geopm
         }
     }
 
-    StreamModelRegion::StreamModelRegion(double big_o_in, int verbosity, bool do_imbalance, bool do_progress, bool do_unmarked)
+    StreamModelRegion::StreamModelRegion(double big_o_in,
```

Comment:
StreamModelRegion ctor needs to be updated to take name as parameter to be used properly by TimedStreamModelRegion.  m_name being updated at 446 below is too late.  Subsequently the name in the report is not "timed_stream" as expected.